src/navigators/createMaterialTopTabNavigator.js

```diff
@@ -15,6 +15,7 @@ import ResourceSavingScene from '../views/ResourceSavingScene';
 type Props = InjectedProps & {
   animationEnabled?: boolean,
   lazy?: boolean,
+  renderLazyComponent?: () => JSX.Element,
```

Comment:
Rename to `renderLazyPlaceholder` for consistency with other `renderX` props (we don't have `Component` prefix)

Reply:
It should be `React.Node` instead of `JSX.Element`

Reply:
I feel like it also needs a way to know the current route so you can render different placeholders for different routes.